src/pages/stacks-blockchain/transactions.md

```diff
@@ -0,0 +1,487 @@
+
+The signing flow for sponsored transactions would be to have the user first sign the transaction with their origin account with the intent of it being sponsored (i.e. the user must explicitly allow a sponsor to sign), and then have the sponsor sign with their paying account to pay for the user's transaction fee.
+
+## Encoding
+
+A transaction includes the following information. Multiple-byte fields are encoded as big-endian.
+
+| **Type**        | **Description**                                                                                                                                                                                                           |
+| --------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Version number  | Network version. `0x80` for testnet, `0x0` for mainnet                                                                                                                                                                    |
+| Chain ID        | Chain instance ID. `0x80000000` for testnet, `0x00000001` for mainnet                                                                                                                                                     |
+| Authorization   | Type of authorization (`0x04` for standard, `0x05` for sponsored) and [spending conditions](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-005-blocks-and-transactions.md#transaction-authorization) |
+| Post-conditions | List of post-conditions, each including a [type ID and varible-length condition body](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-005-blocks-and-transactions.md#transaction-post-conditions-1)   |
+| Payload         | Transaction type and variable-length [payload](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-005-blocks-and-transactions.md#transaction-payloads-1)                                                 |
+
+## Construction
+
+The easies way to construct well-formed transactions is by [using the Stacks Transactions JS library](https://github.com/blockstack/stacks-transactions-js#post-conditions). You can construct the following transaction types:
+
+- Stacks token transfer
+- Smart contract deploy
+- Smart contract function call
+
+When constructing transactions, it is requireq to set the network the transaction is intended for. This can be either mainnet or testnet. At the moment of this writing, the only available option is the [testnet network](/stacks-blockchain/testnet).
+
+-> Transactions can be constructed and serialized offline. Internet access is only required to broadcast the transaction to the network.
```

Comment:
One snag here is that you need to know the up-to-date nonce for the account. Otherwise the transaction might not be valid.